cocos/base/CCEventMouse.h

```diff
@@ -189,6 +196,8 @@ class CC_DLL EventMouse : public Event
     friend class EventListenerMouse;
 };
 
+std::ostream& operator<<(std::ostream& out, const EventMouse::MouseButton& button);
```

Comment:
I think for the moment I'll take it out and leave it to be implemented by anyone that wants it. The implementation is pretty trivial.